configs/upgrade_timeouts.yml

```diff
@@ -0,0 +1,11 @@
+:prepare_repositories: 120
+:pre_upgrade: 300
+:upgrade_os: 1500
+:post_upgrade: 600
+:evacuate_host: 300
+:chef_upgraded: 1200
+:router_migration: 600
+:lbaas_evacuation: 600
+:delete_pacemaker_resources: 300
+:delete_cinder_services: 300
+:wait_until_compute_started: 60
```

Comment:
You're missing at least `reload_nova_services` and `online_migrations` ... and `shutdown_services`, `shutdown_remaining_services`

Reply:
And `delete_nova_services`

Reply:
updated to SOC8 set of timeouts